doc/project/dev_env_and_tools.rst

```diff
@@ -258,138 +258,165 @@ fix it.
+
+This issue is a duplicate of another issue (please specify).
+
+* ``Good first issue``
+
+Good for a first time contributor to take.
+
+* ``Release Notes``
 
-==================== ===========================================================
-``Regression``       Something, which was working, but does not anymore
-                     (bug subtype)
-``Question``         This issue is a question to the Zephyr developers
-``Enhancement``      Changes/Updates/Additions to existing
-                     :ref:`features <feature-tracking>`
-``Feature request``  A request for a new :ref:`feature <feature-tracking>`
-``Feature``          A :ref:`planned feature<feature-tracking>` with a milestone
-``Duplicate``        This issue is a duplicate of another issue
-                     (please specify)
-``Good first issue`` Good for a first time contributor to take
-``Release Notes``    Issues that need to be mentioned in release notes as known
-                     issues with additional information
-==================== ===========================================================
+Issues that need to be mentioned in release notes as known issues with
+additional information.
 
 Any issue must be classified and labeled as either ``Bug``, ``Question``,
 ``Enhancement``, ``Feature``, or ``Feature Request``. More information on how
 feature requests are handled and become features can be found in
 :ref:`Feature Tracking<feature-tracking>`.
+
+Labels applicable to pull requests only
+=======================================
+
+The issue or PR describes a change to a stable API. See additional information
+in :ref:`stable_api_changes`.
+
+* ``Hotfix``
+
+Fix for an issue blocking development.
+
+* ``Trivial``
+
+* ``Maintainer``
+
+Maintainer review reqiured.
+
+* ``Security Review``
+
+To be reviewed by a security expert.
+
+* ``DNM``
+
+This PR should not be merged (Do Not Merge). For work in progress, GitHub
+"draft" PRs are preferred.
+
+* ``Stale``
+
+PR which seems abandoned, and requires attention by the author.
+
+* ``Needs review``
+
+The PR needs attention from the maintainers.
+
+* ``Backport``
+
+The PR is a backport or should be backported.
+
+* ``Licensing``
+
+The PR has licensing issues which require a licensing expert to review it.
+
+Depending on the PR complexity, an indication of how long a merge should be held
+to ensure proper review can found at the section on :ref:`review process <review_time>`.
+
+Labels applicable to both pull requests and issues
+==================================================
+
+* ``Area:*``
+
+Indicates subsystems (e.g., "Area: Kernel", "Area: I2C", "Area: Memory
+Management"), project functions (e.g., "Area: Debugging", "Area: Documentation",
+"Area: Process"), or other categories (e.g., "Area: Coding Style", "Area:
+MISRA-C") affected by the bug or pull request.
+
+An area maintainer should be able to filter by an area label and find all issues
+and PRs which relate to that area.
+
+* ``Platform:*``
```

Comment:
This should be `platform: *`.

Reply:
Fixed in https://github.com/zephyrproject-rtos/zephyr/pull/44774/commits/cab8ea2758bb3f11808fc10b046d7cfa5d797508 .